images/capi/ansible/roles/sysprep/tasks/main.yml

```diff
@@ -30,10 +30,6 @@
     state: absent
   when: http_proxy is defined or https_proxy is defined
 
-- name: set Ubuntu pw
-  command: |-
-    echo 'ubuntu:ubuntu' | chpasswd -c SHA512
```

Comment:
IIRC `ubuntu:ubuntu` is the default pw anyways, correct?
If not then the problem can be that during cloud-init our `dhc-init.sh` script will not be triggered and we're unable to connect to the machine via SSH.

Reply:
This was moved to our internal ansible role, so the unit is not affected

Reply:
So it‘s not just in the script, okay 👍🏻